ci_framework/roles/env_op_images/tasks/main.yml

```
@@ -0,0 +1,92 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure directory is present
+  ansible.builtin.file:
+    path: "{{ cifmw_env_op_images_dir }}"
```

hmm, you don't mention the `{{ item }}` so you'll create twice the exact same directory.
You want
```suggestion
    path: "{{ cifmw_env_op_images_dir }}/{{ item }}"
```